memory/ndctl.py

```diff
@@ -266,8 +274,25 @@ def test_namespace_modes(self):
         if failed_modes:
             self.fail("Namespace for %s mode failed!" % failed_modes)
 
+    def test_multiple_namespaces_region(self):
+        """
+        Test multiple namespace with single region
+        """
+        cnt = self.params.get('namespace_cnt', default=5)
+        self.enable_region()
+        region = self.get_json(short_opt='-R')[0]
+        self.log.info("Using %s for muliple namespace regions", region)
+        self.disable_namespace(region=region)
+        self.destroy_namespace(region=region)
+        size = self.get_json_val(self.get_json(
+            long_opt='-r %s' % region)[0], 'size')
+        self.log.info("Creating %s namespaces", cnt)
+        for nid in range(0, cnt):
+            self.create_namespace(region=region, mode='fsdax', size=size / cnt)
+            self.log.info("Namespace %s created", nid + 1)
```

Comment:
Suggestion: Log can be moved to create_namespace()

Reply:
This is specific to particular test, just to specify the count of namespace.

Reply:
ok